CHANGELOG.md

```diff
@@ -4,6 +4,17 @@
 
 ## [Unreleased] - ReleaseDate
 
+### Fixed
+
+- Do now allow empty spec name.
+- Bail out an error for non remote registry.
+
+### Changed
+
+- Add empty name case.
+- Add a case for inspect the unpublished workspace package.
+- All use ws in tests.
```

Comment:
Wouldn't these be more of "Internal" or something?

Reply:
Removed!